python/tank/bootstrap/resolver.py

```diff
@@ -84,7 +85,17 @@ def resolve_configuration(self, config_descriptor, sg_connection):
             # convert to dict so we can introspect
             config_descriptor = descriptor_uri_to_dict(config_descriptor)
 
-        if config_descriptor["type"] == constants.BAKED_DESCRIPTOR_TYPE:
+        if config_descriptor["type"] == constants.INSTALLED_DESCRIPTOR_TYPE:
+            config_path = config_descriptor["path"]
```

Comment:
this is not the config path, right, but the path that points at the parent folder of the config? Might be worth clarifying.